src/main_menu.py

```diff
@@ -142,6 +143,7 @@ class EndGameWindow:
     """
     Dialog when quitting the game
     """
+
```

Comment:
Bitte nichts in den anderen Dateien ändern

Reply:
Das zeigt komischerweise ständig Änderungen an Dateien an denen ich nicht gearbeitet habe.

Reply:
> Darf ich vielleicht wissen wer @PrimordialSphere ist? 😅

Me!

Reply:
Es geht hier leider mit meinem Account drunter und drüber. Lokal bin ich anscheinend in dem angemeldet.

Reply:
Das wäre toll, wenn du mir das morgen nochmal kurz zeigst. Travis lässt den Code jetzt endlich durch. :)